### ISSUE TITLE
[A11Y] - "Latest synced block" text contract

### ISSUE THREAD

**Issue by josephcoombes** [Uniswap/v3-info#216]
**Bug Description**
<img width="232" alt="image" src="https://user-images.githubusercontent.com/6978295/162485682-4e51015c-993b-4385-856e-141d93ff28f4.png">

Element has insufficient color contrast of 2.79 (foreground color: #6c7284, background color: #2c2f36, font size: 8.3pt (11px), font weight: normal). Expected contrast ratio of 4.5:1

This applies to both the label and green value text.

**Steps to Reproduce**

1. Chromium dev tools might be able to show the contrast ratio. Otherwise an accessibility browser extension such as [axe DevTools](https://chrome.google.com/webstore/detail/axe-devtools-web-accessib/lhdoppojpmngadmnindnejefpokejbdd) will also identify this issue:

**Expected Behavior**
Elements should have a minimum contrast ratio of 4.5:1

Returning the block text to full opacity would resolve this issue for that value:
<img width="641" alt="image" src="https://user-images.githubusercontent.com/6978295/162486196-a371998f-b7f6-46c6-a2e9-ea1ecd9deb4c.png">
